discovery/design/kubernetes.md

```diff
@@ -76,7 +76,7 @@ If additional processing is required, then an argument can be changed to increas
 
 Those labels are defined as:
 
-- gimbal.heptio.com/Cluster: ClusterName (Defined via argument)
+- gimbal.heptio.com/backend: ClusterName (Defined via argument)
```

Comment:
I think we should lowercase all the labels. Backend is now lower but the other's arent.

Reply:
I think the code is fine just the docs have caps.